Makefile

```diff
@@ -18,9 +18,8 @@ SHELL := /bin/bash
 VENV_ACTIVATE := .venv/bin/activate
 
 ${VENV_ACTIVATE}: requirements.txt
-	rm -rf .venv
 	python3 -m venv .venv
-	source ${VENV_ACTIVATE} && python3 -m pip install -r requirements.txt
+	source ${VENV_ACTIVATE} && python3 -m pip install --upgrade -r requirements.txt
```

Comment:
Why did we add this?

Reply:
Ah, sorry the --upgrade flag may not be necessary.